hydromt_delft3dfm/workflows/boundaries.py

```diff
@@ -319,7 +319,7 @@ def compute_2dboundary_values(
     else:
         # prepare boundary data
         # get data freq in seconds
-        _TIMESTR = {"D": "days", "H": "hours", "T": "minutes", "S": "seconds"}
+        _TIMESTR = {"D": "days", "h": "hours", "min": "minutes", "s": "seconds"}
```

Comment:
With regard to the bugfix, this is indeed the fix (I also encountered this myself). Might be good to move this to a global variable as it keeps getting used. Also we do not restrict the pandas version, but this only supports > 2.2.0. Good to also restrict pandas version to > 2.2.0?

Reply:
Also fixes https://github.com/Deltares/hydromt_delft3dfm/issues/134

Reply:
I am pretty sure this is also supported by older pandas versions, could it not be that from 2.2.0 it raises errors instead of futurewarnings? I have been seeing futurewarnings about these lowercase frequencies for quite a while already.